modules/doc/content/newsletter/2024/2024_01.md

```diff
@@ -1,12 +1,29 @@
 # MOOSE Newsletter (January 2024)
 
-!alert! construction title=In Progress
-This MOOSE Newsletter edition is in progress. Please check back in February 2024
-for a complete description of all MOOSE changes.
-!alert-end!
-
 ## MOOSE Improvements
 
+### New Physics action syntax
+
+The [Physics](syntax/Physics/index.md) syntax has been added to MOOSE. The `Physics` system is meant
+to streamline the input of an equation in MOOSE, by selecting good default parameters and preconditioners
+for example. `Physics` are derived from [Actions](syntax/Actions/index.md) but define additional APIs
+to facilitate the implementation of a new `Physics` action. As appropriate, some of the modules' actions that defined equations will be transitioned to the `Physics` system over the coming months.
```

Comment:
```suggestion
The [Physics](syntax/Physics/index.md) syntax has been added to MOOSE. The `Physics` system is meant
to streamline the input of an set of physical equations in MOOSE. For example, this might allow the definition of the required equations automatically but also the selection of good default parameters and preconditioners. `Physics` are derived from [Actions](syntax/Actions/index.md) but define additional APIs
to facilitate the implementation of a new `Physics` action. As appropriate, some of the modules' actions that defined equations will be transitioned to the `Physics` system over the coming months.
```

@GiudGiud